Sources/API/Converter.swift

```diff
@@ -867,82 +791,72 @@ extension Converter {
     /**
      * `toTextNodes` converts the given model to Protobuf format.
      */
-    // TODO: RGATreeSplit is not implemented
-//    static func toTextNodes(rgaTreeSplit: RGATreeSplit<string>) -> [PbTextNode] {
-//        const pbTextNodes = [];
-//        for (const textNode of rgaTreeSplit) {
-//            const pbTextNode = new PbTextNode();
-//            pbTextNode.setId(toTextNodeID(textNode.id));
-//            pbTextNode.setValue(textNode.value);
-//            pbTextNode.removedAt = toTimeTicket(textNode.removedAt));
-//
-//            pbTextNodes.push(pbTextNode);
-//        }
-//
-//        return pbTextNodes;
-//    }
+    static func toTextNodes(_ rgaTreeSplit: RGATreeSplit<TextValue>) -> [PbTextNode] {
+        var pbTextNodes = [PbTextNode]()
+        for textNode in rgaTreeSplit {
+            var pbTextNode = PbTextNode()
+            pbTextNode.id = toTextNodeID(id: textNode.id)
+            pbTextNode.value = String(describing: textNode.value)
+            
+            if let removedAt = textNode.removedAt {
+                pbTextNode.removedAt = toTimeTicket(removedAt)
+            }
+            pbTextNodes.append(pbTextNode)
+        }
+
+        return pbTextNodes
+    }
 
     /**
      * `fromTextNode` converts the given Protobuf format to model format.
      */
-//    function fromTextNode(pbTextNode: PbTextNode): RGATreeSplitNode<string> {
-//        const textNode = RGATreeSplitNode.create(
-//            fromTextNodeID(pbTextNode.getId()!),
-//            pbTextNode.value,
-//        );
-//        textNode.remove(fromTimeTicket(pbTextNode.removedAt));
-//        return textNode;
-//    }
+    static func fromTextNode(_ pbTextNode: PbTextNode) -> RGATreeSplitNode<TextValue> {
+        let textNode = RGATreeSplitNode(fromTextNodeID(pbTextNode.id), TextValue(pbTextNode.value))
+        textNode.remove(pbTextNode.hasRemovedAt ? Self.fromTimeTicket(pbTextNode.removedAt) : nil)
+        return textNode
+    }
 }
 
 // MARK: RichTextNode
 extension Converter {
     /**
      * `fromRichTextNode` converts the given Protobuf format to model format.
      */
-    // TODO: RGATreeSplitNode is not implemented
-//    static func fromRichTextNode(_ pbTextNode: PbRichTextNode) -> RGATreeSplitNode<RichTextValue> {
-//        const richTextValue = RichTextValue.create(pbTextNode.value);
-//        pbTextNode.getAttributesMap().forEach((value) => {
-//            richTextValue.setAttr(
-//                value.key,
-//                value.value,
-//                fromTimeTicket(value.getUpdatedAt())!,
-//            );
-//        });
-//
-//        const textNode = RGATreeSplitNode.create(
-//            fromTextNodeID(pbTextNode.getId()!),
-//            richTextValue,
-//        );
-//        textNode.remove(fromTimeTicket(pbTextNode.removedAt));
-//        return textNode;
-//    }
+    static func fromRichTextNode(_ pbTextNode: PbRichTextNode) -> RGATreeSplitNode<TextValue> {
```

Comment:
Is there any reason why both `fromRichTextNode` and `fromTextNode` should exist?
We discussed removing `Text` and changing `RichText` to `Text` before.
https://github.com/yorkie-team/yorkie/issues/435

Reply:
We haven't decided how to refactor richtext and text in protobuf yet, so the code remains. It will be organized after the server is integrated.